packages/vike-node/README.md

```diff
@@ -108,40 +108,28 @@ export default {
 }
 ```
 
-## Caching and compression:
+## Compression:
 
-In production, `vike-node`:
+In production, `vike-node` compresses all Vike responses
 
-- compresses all Vike responses
-- caches the compressed static assets(.js, .css).
-
-On a request, if the asset(.js, .css) is not in the cache, `vike-node` compresses it with a fast compression level, sends it in the response, then recompresses it with a high compression level and finally caches the compressed data.<br>
-You can disable compression/caching:
+On a request, `vike-node` compresses any asset with a fast compression level, and sends it in the response.<br>
+You can disable compression:
 
 ```js
 app.use(
   vike({
-    compress: false,
-    static: {
-      cache: false
-    }
+    compress: false
   })
 )
 ```
 
 ## Custom [pageContext](https://vike.dev/pageContext):
 
-You can define custom [pageContext](https://vike.dev/pageContext) properties:
+`vike-node` leverages [universal-middleware](https://universal-middleware.dev/) internally,
+and merges the universal context with `pageContext`.
 
-```js
-app.use(
-  vike({
-    pageContext: (req) => ({
```

Comment:
How about we keep this? For simple(/most?) use cases it's easier than having to create a universal context middleware. We can add a note mentioning `universal-middleware`.

Reply:
This one was tricky. I removed it because it made plugging `universal-middleware` more complex (because of the now unnecessary `PlatformRequest` that was passed all around).

With this options as it is before this PR, you can only read `req.*` and feed it to Vike's _pageContextInit_. Supposing that with `express` you were doing something like `req.user = ...`, you could retrieve it here.
Now with `universal-middleware`, you do not edit the original `req` directly anymore, you can only return a Context, edit the web-standard `Request` (or return a `Response`, but irrelevant here). The Context is already fed to _pageContextInit_, as well as `originalHeaders` and `url`. So I'm not seeing anything else relevant that can be read by the user and fed to `pageContext` from the `Request`.

Reply:
Ok :+1:

Actually https://github.com/vikejs/vike/issues/1268 will remove/alleviate the need to set `pageContextInit`.

Also, the upcoming Vike hook `+middleware` will make it a bit easier to create universal middlewares (as it automatically attaches it to the server).

One small thing: the page https://universal-middleware.dev/recipes/context-middleware is neat but `import contextMiddleware from "@universal-middleware-examples/tool/middlewares/context-middleware-hono"` is misleading. It took me a little while to understand that it refers to the code snippet above (I guess the code snippets are copied from real examples).

Reply:
> One small thing: the page https://universal-middleware.dev/recipes/context-middleware is neat but `import contextMiddleware from "@universal-middleware-examples/tool/middlewares/context-middleware-hono"` is misleading. It took me a little while to understand that it refers to the code snippet above (I guess the code snippets are copied from real examples).

They do, I can make it clearer :+1: